backend/services/mgmt/v1alpha1/connection-service/connection.go

```diff
@@ -39,17 +39,99 @@ func (s *Service) CheckConnectionConfig(
 
 	cctx, cancel := context.WithTimeout(ctx, 5*time.Second)
 	defer cancel()
-	err = db.PingContext(cctx)
+	// err = db.PingContext(cctx)
+	tx, err := db.BeginTx(cctx, &sql.TxOptions{ReadOnly: true})
```

Comment:
why does this need to happen inside a transaction?

Reply:
i guess it doesn't, was just being careful and looking at a reference. it can just call it directly